networking/network_observability/observing-network-traffic.adoc

```diff
@@ -0,0 +1,9 @@
+:_content-type: ASSEMBLY
+[id="nw-observe-network-traffic"]
+= Observing the network traffic
+include::_attributes/common-attributes.adoc[]
+:context: nw-observe-network-traffic
+
+toc::[]
+
```

Comment:
I suggest reviewing [assembly guidelines](https://redhat-documentation.github.io/modular-docs/#assembly-guidelines). You need an assembly introduction before the modules. 

This could be a great way to insert some of the content from your module, too.

Reply:
Forgot to mention that this file is still WIP. I had to push this assembly for preview purposes.